recipe/meta.yaml

```diff
@@ -1,43 +1,53 @@
-{% set version = "1.5.0" %}
+{% set name = "geoviews" %}
+{% set version = "1.5.1" %}
 
 package:
-  name: geoviews
+  name: {{ name }}
```

Comment:
Defining a variable that will be used only once is adding a line and obfuscating the recipe for no real gain. Do you mind reverting this change?

Reply:
I have no problem reverting this.

(I copied it from conda-forge's docs at https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml, but I guess there `name` is also used in the download link.)

Reply:
Yeah, that makes sense only when copying and pasting to create a new recipe, which we don't do anymore thanks to `conda-skeleton`. It is a relic that only obfuscate the recipe.